Frameworks/UIKit/UIButton.mm

```diff
@@ -719,6 +747,24 @@ - (void)touchesMoved:(NSSet*)touchSet withEvent:(UIEvent*)event {
     [super touchesMoved:touchSet withEvent:event];
 }
 
+// Updates our highlighted state as needed
+- (void)_updateHighlightedState {
+    [self setHighlighted:(_isInTouchSequence || _isXamlElementPressed)];
```

Comment:
good point; we shouldn't need this extra variable

---
In reply to: [107766623](https://github.com/Microsoft/WinObjC/pull/2309#discussion_r107766623) [](ancestors = 107766623)